mobile/src/bundles/common/components/selector/selector.tsx

```diff
@@ -0,0 +1,114 @@
+import { type ReactElement } from 'react';
+import React from 'react';
+
+// import Icon from 'react-native-vector-icons/MaterialIcons';
+import {
+    FlatList,
+    Pressable,
+    Text,
+    TouchableOpacity,
+    View,
+} from '~/bundles/common/components/components';
+
+import { Color } from '../../enums/enums';
+import { globalStyles } from '../../styles/styles';
+import { styles } from './styles';
+
+type Select = {
+    label: string;
+    value: string;
+};
+
+type Properties = {
+    label: string;
+    options: Select[];
+    onSelect?: (item: Select) => void;
+};
+
+const Selector: React.FC<Properties> = ({ label, options }) => {
+    const [chooseData, setChooseData] = React.useState('Option');
+    const [isModalVisible, setIsMOdalVisible] = React.useState(false);
+
+    const DropdownButton = React.useRef<View>(null);
+
+    const changeModalVisibility = (): void => {
+        setIsMOdalVisible((previous) => !previous);
+    };
+
+    const onPressItem = (option: Select): void => {
+        changeModalVisibility();
+        setChooseData(option.label);
+    };
+
+    return (
+        <View>
+            <Text
+                category="Label"
+                style={{ fontWeight: '500', color: Color.TEXT }}
```

Comment:
```suggestion
                category={TextCategory.LABEL}
```
Category "Label" should already have needed styles

Reply:
TextCategory.LABEL has FontFamily.INTER_REGULAR, but in Figma, the font-weight is 500.

Reply:
Yes, that is my fault. I just double-checked and for Label it should be 500. Sorry for that. 
I think you can change src/bundles/common/styles/fonts.ts 
```
[TextCategory.LABEL]: {
        fontFamily: FontFamily.INTER_MEDIUM,
        fontSize: 13,
        color: Color.TEXT,
    },
```